samples/server.c

```diff
@@ -536,12 +536,23 @@ int main(int argc, char *argv[])
             if (irq_triggered) {
                 irq_triggered = false;
                 vfu_irq_trigger(vfu_ctx, 0);
-
+                /*
+                 * Apart from triggering an IRQ via the eventfd, we also
+                 * trigger an IRQ via a message, simply for demonstrating how
+                 * it's done. The client expects this behavior from the server.
+                 */
                 ret = vfu_irq_message(vfu_ctx, 0);
                 if (ret < 0) {
                     err(EXIT_FAILURE, "vfu_irq_message() failed");
                 }
 
+                /*
+                 * We also initiate some dummy DMA via an explicit message,
+                 * again to show how DMA is done. This is used if the client's
+                 * RAM isn't mappable or the server implementation prefers it
+                 * this way.  Again, the client expects the server to send DMA
+                 * messages right after it is has triggered the IRQs.
```

Comment:
one more round, s/it is/it/

Reply:
sorry, brain isn't working today

Reply:
if it takes 3 rounds to review a PR with only comments then how long will it take for the PRs with code changes ? xD